scompose/project/project.py

```diff
@@ -64,8 +64,8 @@ def set_filename(self, filename):
         ==========
         filename: the singularity-compose.yml file to use
         """
-        self.filename = filename or "singularity-compose.yml"
-        self.working_dir = os.path.dirname(os.path.abspath(self.filename))
+        self.filename = filename
```

Comment:
I would leave the default here to "singularity-compose.yml" - and if you are changing this to require a list, then please support the old behavior to still provide a single filename, and have it checked if it's a list, single string, or None, and process accordingly. And then if your preference is to store in a list, the variable should be self.filenames to indicate plural / multiple files. 

And if I understand correctly, the working directory is changed to os.getcwd() to account for possibly different compose files in different directories. Of course the issue here is that all compose files need to then have their context relevant to that directory (is that your intention?)

Reply:
yes correct! We wouldn't want to break previous scripts that are providing a string. The simple solution I think is to allow either.